config/routes.rb

```diff
@@ -1,3 +1,14 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  mount RailsAdmin::Engine => '/admin', as: 'rails_admin'
+
+  devise_for :users, :controllers => {:registrations => "registrations"}
```

Comment:
`devise_for :users, controllers: { registrations: 'registrations' }`
decide if you use double quotes or single quotes